### PR TITLE
Update README.md and .env-sample (#79)

### PR DESCRIPTION
**Pull request by jvaclavik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,33 +11,57 @@ This project is a living example of a client app you could start building with T
 To get started, clone the repository, change directory and install the dependencies:
 
 ```shell
-git clone git@github.com:kiwicom/margarita.git
+https://github.com/kiwicom/margarita
```
Comment (JoseDuSV) [inline README.md]: Why not `git clone git@github.com:kiwicom/margarita.git` ?

Comment (jvaclavik) [reply README.md]: `https` doesn't need any authentication which seems to me better in this case. But it's not a big deal, we can keep the old one.

Comment (JoseDuSV) [reply README.md]: Ok, thanks, that makes sense. But then it should be at least `git clone https://github.com/kiwicom/margarita`

Comment (jvaclavik) [reply README.md]: I see, you are right! Thank you. It's copy-paste issue

Comment (jvaclavik) [reply README.md]: Now it should be fixed

**Pull request merged**